tool/data/config/metrics_test.go

```diff
@@ -16,7 +16,8 @@ import (
 func TestMetrics_ToMap(t *testing.T) {
 	expectedKey := "metrics"
 	expectedValue := map[string]interface{}{
-		"append_dimensions": map[string]interface{}{"ImageId": "${aws:ImageId}", "InstanceId": "${aws:InstanceId}", "InstanceType": "${aws:InstanceType}", "AutoScalingGroupName": "${aws:AutoScalingGroupName}"},
+		"aggregation_dimensions": [][]string{{"InstanceId"}},
```

Comment:
I not sure if this part is properly formatted.

Reply:
I agree that it looks off, but when I run `make fmt`, this is the way it's formatted.

Reply:
ok, there is no way to tweak gofmt anyway 😄